pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts =
+  --doctest-modules
+  --ignore=setup.py
```

Comment:
you can also use `testpaths` to list the directories that should be searched for tests:
https://pytest.org/latest/customize.html#confval-testpaths

E.g.:

```
testpaths =
    Lib/fontParts
    test
```